package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "eslint --ignore-path .gitignore --ext ts,tsx,js . && stylelint --ignore-path .gitignore '{src,pages}/**/*.{css,scss}'",
     "start": "gulp watch & NODE_ENV=development webpack serve --config pages/webpack.config.js",
     "start:integ": "NODE_ENV=development webpack serve --config pages/webpack.config.integ.js",
-    "preinstall": "./scripts/prepare-package-lock.js",
+    "postinstall": "node ./scripts/unlock-package-lock.js",
```

Comment:
There is no longer a true preinstall script. Using postinstall here to be more clear about when this is running.

Reply:
Why do we need to run it in CI at all? Wouldn't https://github.com/cloudscape-design/.github/pull/34 change be enough to solve the task?

Reply:
This is for developers for when they run install and if they don't do a commit where the package-lock can get linted then the package versions would be locked the next time they try to pull the latest updates. Unless I'm just misunderstanding npm here?

Reply:
I am not sure about lifecycle order here. I recall npm wrote the lockfile after our script runs.

I can test it again after merging this

Reply:
Hmm, you're right here actually. In < npm8 we used to hook into post shrinkwrap to be able to modify the package-lock 🤔 So then yes this line becomes useless and can be removed.